Sources/GRPC/CallHandlers/_BaseCallHandler.swift

```diff
@@ -23,13 +23,8 @@ import Logging
 ///
 /// Calls through to `processMessage` for individual messages it receives, which needs to be implemented by subclasses.
 /// - Important: This is **NOT** part of the public API.
-public class _BaseCallHandler<RequestPayload: GRPCPayload, ResponsePayload: GRPCPayload>: GRPCCallHandler {
-  public func makeGRPCServerCodec() -> ChannelHandler {
-    return HTTP1ToGRPCServerCodec<RequestPayload, ResponsePayload>(
-      encoding: self.callHandlerContext.encoding,
-      logger: self.logger
-    )
-  }
+public class _BaseCallHandler<Request, Response>: GRPCCallHandler {
+  public let codec: ChannelHandler
```

Comment:
Should this also be underscored, for consistency? We don't actually want users touching it.